docs/scos/dev/back-end-development/zed-ui-tables/create-and-configure-zed-tables.md

```diff
@@ -129,6 +129,25 @@ $config->setDefaultSortDirection(
     \Spryker\Zed\Gui\Communication\Table\TableConfiguration::SORT_DESC);
 ```
 
+### Configure strict search
+
+Fuzzy search is the default search option in Back Office data tables. Fuzzy search makes use of the SQL logical operator ‘LIKE’ in combination with ‘LOWER’ for comparison. This search option may result in performance issues on larger tables due to indexes not being used.
```

Comment:
```suggestion
The default search option in Back Office data tables searches for anything that contains the specified sub strings. This default search makes use of the SQL logical operator ‘LIKE’ in combination with ‘LOWER’ for comparison. It may result in performance issues on larger tables due to indexes not being used.
```